qa/tasks/cephfs/test_volumes.py

```diff
@@ -302,5 +356,27 @@ def test_nonexistent_subvolume_group_snapshot_rm(self):
         # remove subvolume
         self._fs_cmd("subvolume", "rm", self.volname, subvolume, group)
 
+        # verify trash dir is clean
+        self._wait_for_trash_empty()
+
         # remove group
         self._fs_cmd("subvolumegroup", "rm", self.volname, group)
+
+    def test_async_subvolume_rm(self):
```

Comment:
When I run the tests locally (with vstart_runner on vstart cluster), this test is timing out even if the number of files is as low as 10.
```
2019-06-21 22:25:39,396.396 INFO:__main__:Running ['getfattr', '--only-values', '-n', 'ceph.dir.entries', '/tmp/tmpg2wMfb/mnt.0/./volumes/_deleting']
2019-06-21 22:25:40,421.421 INFO:__main__:test_async_subvolume_rm (tasks.cephfs.test_volumes.TestVolumes) ... ERROR
2019-06-21 22:25:40,422.422 ERROR:__main__:Traceback (most recent call last):
  File "/home/rraja/git/ceph/qa/tasks/cephfs/test_volumes.py", line 382, in test_async_subvolume_rm
    self._wait_for_trash_empty()
  File "/home/rraja/git/ceph/qa/tasks/cephfs/test_volumes.py", line 69, in _wait_for_trash_empty
    self.mount_a.wait_for_dir_empty(trashdir)
  File "/home/rraja/git/ceph/qa/tasks/cephfs/mount.py", line 255, in wait_for_dir_empty
    i, dirname, self.client_id))
RuntimeError: Timed out after 30s waiting for ./volumes/_deleting to become empty from 0
```
We'll need to check if this test passes in teuthology